packages/react-aria-components/src/ComboBox.tsx

```diff
@@ -47,10 +47,16 @@ export interface ComboBoxRenderProps {
    * Whether the combobox is required.
    * @selector [data-required]
    */
-  isRequired: boolean
+  isRequired: boolean,
+  // TODO: do we want loadingState for RAC Combobox or just S2
+  /**
+   * Whether the combobox is currently loading items.
+   * @selector [data-loading]
+   */
+  isLoading?: boolean
```

Comment:
For now see about moving `isLoading` to the loading spinner? Otherwise, just make sure the followup refactor goes in before next release